### PR TITLE
307 Make your vote count

### PR DESCRIPTION
**Pull request by dilbertKocik**
The purpose of this rule is to make it clear how the votes will be counted. It is understood that the wording of this rule allows a player to change their vote.

### CODE REVIEW

File: laws.md [hunk 1]
```diff
@@ -70,3 +70,5 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 It is important, for parsing, that no other non-space characters be included in the comment body. The comment must be attached to the PR itself, and not associated with any specific file.
 
 **305.** No player may be a judge in a dispute in which they are involved.
+
+**306.** PARSING VOTES: Only the most recent vote will be counted. All other votes by a given player will be ignored.
```
Comment (ArchmageInc) [inline laws.md]: This should be 307 as there is an active proposal for 306. Even if 306 is declined, this would still be 307 as I understand.

**Comment by jimmyhmiller** [comment]
I like the idea. Once we get into bots we will have to figure out how to make sure they stay in compliance with the rules.

And I agree with Brandon it should be 307.

**Comment by dilbertKocik** [comment]
This proposal is open for voting.

**Comment by dilbertKocik** [comment]
yay

**Comment by jimmyhmiller** [comment]
Yay

**Comment by ArchmageInc** [comment]
nay

**Comment by jimmyhmiller** [comment]
Trying to get lots of points ArchmageInc?

**Comment by ArchmageInc** [comment]
Quorum Reached, the Motion Passes. I get 10 points, Robert gets 16

**Pull request merged**